scripts/e2e.mosaic.sh

```diff
@@ -31,7 +31,7 @@ echo "Installing updated web3 via virtual registry "
 echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
 
 git submodule update --init --recursive
-yarn --registry http://localhost:4873
+yarn --registry http://localhost:4873 --network-timeout 100000
```

Comment:
Sorry, I'm sneaking this into this PR because I forgot about this yarn command in #3322 yesterday and the network connectivity problem remains on Travis today.